### PR TITLE
Blog draft

### PR DESCRIPTION
**Pull request by ramumaha**
The blog background doesn't seem to extend through the page when we add the card components to it.As I have used the same card component for both blog and events the changes are reflected in both.Is it fine or do you want me to create a new component for each?

### CODE REVIEW

File: src/pages/2020_2021.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+          borderRadius="lg"
+        >
+          <Box w={{ lg: "45%" }}>
+            <Box
+              h={{ base: 14, sm: "full" }}
+              rounded={{ lg: "lg" }}
+              bgSize="cover"
+              bgImage={vvcoding}
+              width={300}
+              bgSize="contain"
```
Comment (harisarang) [inline src/pages/2020_2021.js:34]: Duplicate props here

File: src/pages/2020_2021.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
```
Comment (harisarang) [inline src/pages/2020_2021.js:88]: Duplicate props here

File: src/pages/blog.js [hunk 3]
```diff
@@ -1,8 +1,73 @@
 import React from "react";
-import { Box, Button } from "@chakra-ui/react";
+import { Container, Stack, Box, Heading, Text, Img } from "@chakra-ui/react";
```
Comment (harisarang) [inline src/pages/blog.js:2]: Unused imports

File: src/pages/past.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+import { render } from "react-dom";
+class Past extends Component {
+  render() {
+    return (
+      <Flex p={30} w="full" alignItems="center" justifyContent="left">
+        <EventHeader />
+
+        <Box
+          as="span"
+          //  bg={useColorModeValue("white", "gray.900")}
```
Comment (harisarang) [inline src/pages/past.js:14]: If the comments are not going to be used it can be removed

File: src/pages/past.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+          borderRadius="lg"
+        >
+          <Box w={{ lg: "45%" }}>
+            <Box
+              h={{ base: 14, sm: "full" }}
+              rounded={{ lg: "lg" }}
+              bgSize="cover"
+              bgImage={vvcoding}
+              width={300}
+              bgSize="contain"
```
Comment (harisarang) [inline src/pages/past.js:33]: Look what I hvae found 🧐. Duplicate props 😆

File: src/pages/upcoming.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from "react";
+import { Box, Button, Flex } from "@chakra-ui/react";
```
Comment (harisarang) [inline src/pages/upcoming.js:2]: Remove unused components

**Review by harisarang — CHANGES_REQUESTED** [review]
Make sure not have duplicate props

File: src/pages/past.js [hunk 7]
```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import { chakra, Box, Flex, useColorModeValue, Link } from "@chakra-ui/react";
+import vvcoding from "../images/vvvcoding.svg";
+import EventHeader from "../components/EventHeader";
+import { render } from "react-dom";
```
Comment (harisarang) [inline src/pages/past.js:5]: Remove unused components

File: src/pages/blog.js (hunk 3 above)
Comment (ramumaha) [reply src/pages/blog.js:2]: Removed

File: src/pages/upcoming.js (hunk 6 above)
Comment (ramumaha) [reply src/pages/upcoming.js:2]: removed

**Review by harisarang — APPROVED** [review]

**Pull request merged**